src/shims/unix/fd.rs

```diff
@@ -58,6 +62,19 @@ pub trait FileDescription: std::fmt::Debug + Any {
         // so we use a default impl here.
         false
     }
+
+    // Check the readiness of epoll-supported file description.
+    fn check_and_update_readiness<'tcx>(
+        &self,
+        _ecx: &mut MiriInterpCx<'tcx>,
+    ) -> InterpResult<'tcx> {
+        throw_unsup_format!("{}: epoll does not support this file description", self.name());
+    }
+    fn get_epoll_events<'tcx>(
```

Comment:
Please add a doc comment, explaining in particular the return value. `(i32, i32)` is a very non-descriptive type, that should probably have an alias/newtype that explains its purpore.